docs/rfc/active/000-internationalization.md

```diff
@@ -0,0 +1,165 @@
+- Start Date: (2022-09-14)
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Discussion Issue: (None)
+- Implementation PR(s): (leave this empty)
+
+# Internalization (i18n)
+
+## Summary
+> Implements the internalization of Datahub.
+> Being able to change and add language without modifying the source.
+
+## Basic example
+```javascript
+// Instead of this
+<Button type="submit">
+    Next
+</Button>
+// We write this
+<Button type="secondary">
+    {t('common.next')}
```

Comment:
I like this approach - using a method to resolve the real text inline,  as opposed to introducing a new component into the dom hierarchy

Reply:
In most case we use it, but when you want to interprete html in the translation we have to use <Trans .../>
Example : Using [HTML Text Formatting](https://www.w3schools.com/html/html_formatting.asp) in translated text. (Datahub text often use the <b>bold</b> HTML Element)